docs/content/topics/operators/deploy.md

```diff
@@ -11,4 +11,429 @@ aliases:
 
 # Deploying and managing Brigade
 
-TODO
\ No newline at end of file
+In this doc, we'll go over steps for setting up and managing a full,
+production-grade Brigade deploy.  This includes:
+
+  - [Managing external access to Brigade via a DNS hostname and ingress resource](#managing-inbound-traffic-via-ingress)
+  - [Utilizing a third-party certificate provider for public-facing TLS](#certificates-and-tls)
+  - [Choosing a third-party authentication provider for managing user auth](#third-party-authentication)
+
+(Alternatively, if you're interested in using Brigade in a local development
+environment or trial capacity, take a look at the [QuickStart].)
+
+[QuickStart]: /intro/quickstart.md
+
+## Prerequisites
+
+* A [Kubernetes cluster], version 1.18.8 or higher.
+  Your cluster must be accessible to the outside world, e.g. to third-party
```

Comment:
Third party authn doesn't need the API server to be reachable by anything other than your own web browser. This is a requirement for the gateways more than anything else.